dist/tools/ng_sniffer/README.md

```diff
@@ -0,0 +1,72 @@
+# RIOT Sniffer Application
+
+
+## About
+
+This sniffer application can be used to sniff network traffic using RIOT based
+nodes. It is primarily designed for sniffing wireless data traffic, but can also
+well be used for wired network traffic, as long as the used network devices
+support promiscuous mode and output of raw data.
+
+The sniffer is based on a RIOT node running the [ng_sniffer application](https://github.com/RIOT-OS/applications/tree/master/ng_sniffer) application located in [RIOTs application repository](https://github.com/RIOT-OS/applications).
+This node outputs received network traffic via a serial port in the Wireshark
+pcap format. This output is then parsed by the `ng_sniffer.py` script included
+in this folder run on a host computer.
+
+The `ng_sniffer.py` script is a modified version of [malvira's script](https://github.com/malvira/libmc1322x/blob/master/tools/rftestrx2pcap.py) for the Redbee Ecotag
+(https://github.com/malvira/libmc1322x/wiki/wireshark).
+
+
+## Dependencies
+
+The `ng_sniffer.py` script needs [pyserial](https://pypi.python.org/pypi/pyserial).
+
+Installing the dependencies:
+
+#### Debuntu
+    apt-get install python-serial
+
+#### PIP
+    pip install pyserial
+
+
+## Usage
+
+General usage:
+
+1. Flash an applicable RIOT node with the sniffer application from
+(https://github.com/RIOT-OS/applications/tree/master/ng_sniffer)
+
+2. Run the `ng_sniffer.py` script
+```
+$ ./ng_sniffer.py <tty> <baudrate> <channel> [outfile]
+```
+
+### Examples
+
+The following examples are made when using the sniffer application together with
+an `iot-lab_M3` node that is connected to /dev/ttyUSB1 (or COM1) and runs per
+default with a baudrate of 500000.
```

Comment:
what do you mean? All iot-labs are per dafault configured to 500k baud. Most other boards in RIOT are configured to 115200.